### DISCUSSION TITLE
--restart-from last

### DISCUSSION THREAD

**Discussion by INFINITY1993** [ablab/spades#946] [Q&A] [answered]
Hi, there,

I have finished the metaspades assembly with default kmer set (21, 33 and 55), but now I also want to check the assembly results with kmer set (21,33,55 and 77). Could I use "--restart-from last" options ? will it start from kmer 77?

Best regards!

**Comment by asl** [comment, marked as answer]
Well, no, because "last" is k=55 in your case. You need to explicitly specify the k-mer set and the k-mer to restart from.

**Reply by INFINITY1993** [reply]
Thanks for your quick response.
spades.py \
-o /data/home/soil_metagenome_anaysis/metaspades_assembly_out/s2 \
--meta \
-1 /data/home/soil_metagenome_anaysis/cleandata/s2_1.fastq \
-2 /data/home/soil_metagenome_anaysis/cleandata/s2_2.fastq \
-t 60 \
-m 500 \
--restart-from k55 \
-k 21,33,55,77 \
> metaspades_s2.log 2>&1

Is the settings right? Then it will run  k55  again, or it will skip k55 and run k77?

**Reply by asl** [reply]
It will re-run k=55, as the last stage is always very special.

**Reply by INFINITY1993** [reply]
ok, I see. Thanks so much!
